attestation/attestation.schema.json

```diff
@@ -1,11 +1,19 @@
 {
-  "$id": "ENDcMNUZjag27T_GTxiCmB2kYstg_kqipqz39906E_FD",
-  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "EJH39k_xxuXt5SKfsLYKdx08mnqy8fSUEPtpil6RabsE",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
   "title": "Data Attestation Credential",
```

Comment:
Attestation and Credential are opposites. What you have here is an Attestation. Please don't call it a credential.

Reply:
updated as per comment